### PR TITLE
PUBDEV-8727 - make parameter algorithm of train_segments function in R independet from small/large lettters in algo name

### PR DESCRIPTION
**Pull request by valenad1**
https://h2oai.atlassian.net/browse/PUBDEV-8727

IMHO this change should work and it's backwards compatible.

### CODE REVIEW

**Comment by michalkurka** [comment]
Great idea, this will make it more convenient for users.

**Review by michalkurka — APPROVED** [review]

File: h2o-r/h2o-package/R/segment.R [hunk 1]
```diff
@@ -41,9 +41,13 @@ h2o.train_segments <- function(algorithm,
                                ...)
 {
     train_segments_fun_name <- sprintf(".h2o.train_segments_%s", algorithm)
-  if (!exists(train_segments_fun_name)) {
+  if (!exists(train_segments_fun_name) & !exists(tolower(train_segments_fun_name))) {
```
[the remark this answers is not in the document]
Comment (valenad1) [reply h2o-r/h2o-package/R/segment.R:44]: Thank you, yep just tolower should be good enough :)

**Comment by valenad1** [comment]
@michalkurka Can this PR go to zumbo?

**Comment by michalkurka** [comment]
@valenad1 I would keep targeting master, it is not a fix per se

**Pull request merged**